src/components/App/App.jsx

```diff
@@ -1,5 +1,29 @@
+import logoIconik from "../../assets/images/logo.png";
+import './app.css';
+
 const App = () => (
-  <h1>El salon se llama iconik</h1>
+  <>
+    <section className="menu">
+      <img className='menu__logoIconik' src={logoIconik} alt="Iconik" />
+      <div className="menu__container">
+        <span className="menu__container-item">Dashboard</span>
+        <span className="menu__container-item">Ingresos</span>
+        <span className="menu__container-item">Egresos</span>
+        <span className="menu__container-item">Trabajadores/as</span>
+        <span className="menu__container-item">Clientes</span>
+        <span className="menu__container-item">Citas</span>
+        <span className="menu__container-item">Productos</span>
+        <span className="menu__container-item">Configuracion</span>
```

Comment:
this className `menu__container-item` will be for a link, so please change it to `menu__link`